west.yml

```diff
@@ -118,7 +118,7 @@ manifest:
       revision: 38c79a447e4a47d413b4e8d34448316a5cece77c
       path: modules/debug/segger
     - name: sof
-      revision: 9e75cce8bc1c649672931a4d772b2411a890804b
+      revision: pull/6/head
```

Comment:
not sure this is more intuitive than just a SHA1, but apparently it works

Reply:
this is just pointing to an open PR, once the PR is merged, it will need to move to a SHA.